moon/controller.py

```diff
@@ -232,7 +237,7 @@ def on_rot(self, data):
 
         if self.use_gimbal:
             # maintain camera level
-            cam_angle = self.camera_angle + self.pitch
+            cam_angle = min(math.pi / 4.0, max(-math.pi / 8.0, self.camera_angle + self.pitch))
```

Comment:
why to have duplicate of this code? Is there a reason not to call directly `set_cam_angle()`?

Reply:
set_cam_angle() is meant for discrete commands that change the tilt based on driving purpose. They are implemented via ReqRep as they have to get there. Gimbal-triggered tilt updates are sent at 50Hz using PubSub (no harm if something gets lost).